src/app/signup/page.tsx

```diff
@@ -32,20 +32,23 @@ export default function SignUp() {
     if (e.target.files && e.target.files[0]) {
```

Comment:
In future we can move the image getting part into a common place as it is used in 2 places. Will merge now but going forward keep in mind that if we using any thing more than once then please add it to lib or any other common place